tools/keytools/sign.c

```diff
@@ -22,9 +22,11 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
  */
 
+/* Beware of wolfCrypt user settings in [WOLFBOOT_ROOT]/include/user_settings.h */
+
 /* Option to enable sign tool debugging */
-/* Must also define DEBUG_WOLFSSL in user_settings.h */
-//#define DEBUG_SIGNTOOL
+/* Must also define DEBUG_WOLFSSL in /tools/keytools/user_settings.h */
+/* #define DEBUG_SIGNTOOL */
```

Comment:
This should be a proper options via tools/config.mk, parsed via tools/keytools/Makefile. You can have a separate "config" in your IDE magic perhaps for first run, while Makefile users can add DEBUG_SIGNTOOL=1 in .config

user_settings.h is not meant for being modified.